### PR TITLE
 [RELAY][OPS]LRN and L2_Normalize

### PR DESCRIPTION
**Pull request by siju-samuel**
#1799

- [x] LRN
- [x] L2_NORM 

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from others in the community.

### CODE REVIEW

**Comment by siju-samuel** [comment]
@tqchen @srkreddy1238  @junrushao1994 @MarisaKirisame please review.

File: python/tvm/relay/op/nn/nn.py [hunk 1]
```diff
@@ -364,3 +364,66 @@ def batch_flatten(data):
...
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data to the operator.
+
+    size : int, optional
+        The size of the local region to be considered for normalization.
+
+    axis : int, optional
+        input data layout channel axis. default value is 1 for NCHW format
```
Comment (MarisaKirisame) [inline python/tvm/relay/op/nn/nn.py]: Captialize

File: include/tvm/relay/attrs/nn.h [hunk 2]
```diff
@@ -173,6 +173,44 @@ struct UpSamplingAttrs : public tvm::AttrsNode<UpSamplingAttrs> {
...
+  IndexExpr axis;
+  double bias;
+  double alpha;
+  double beta;
+
+  TVM_DECLARE_ATTRS(LRNAttrs, "relay.attrs.LRNAttrs") {
+    TVM_ATTR_FIELD(size).set_default(5)
+      .describe("The size of the local region to be considered for normalization.");
+    TVM_ATTR_FIELD(axis).set_default(1)
+      .describe("Input data layout channel axis");
```
Comment (MarisaKirisame) [inline include/tvm/relay/attrs/nn.h]: Axis of input data layout channel?

**Comment by tqchen** [comment]
l2 normalization is not as common and let us move it to level2

**Pull request merged**